internal/internal_worker.go

```diff
@@ -865,6 +866,23 @@ func (aw *aggregatedWorker) Start() error {
 		}
 	}
 
+	if !isInterfaceNil(aw.shadowWorker) {
+		if err := aw.shadowWorker.Start(); err != nil {
```

Comment:
Should it stop other workers regardless the error?

Reply:
Sorry, I don't quite follow. Currently, all other workers will be stopped if there's any error on start up.

Reply:
Oh, I see. Should it stop other workers if the shadow worker starts?

Reply:
Yes, it's already doing it. Check [L1274](https://github.com/uber-go/cadence-client/pull/1064/files#diff-81f3fd107107ea44dc42925f83354a318280b38ddb7223548848cfc436c1da58R1274) below.